tests/cases/conformance/es2017/useTypedArrays1.ts

```diff
@@ -0,0 +1,12 @@
+// @target: es5

```

Comment:
for future reference, no need for the tests for changes to lib like these.

Reply:
I took it from neighbor files in `tests/cases/conformance/es2017` but I will keep it on mind